specs/logged_in_users.table

```diff
@@ -8,5 +8,9 @@ schema([
     Column("time", INTEGER, "Time entry was made"),
     Column("pid", INTEGER, "Process (or thread) ID"),
 ])
+extended_schema(WINDOWS, [
+    Column("sid", TEXT, "User security identifier"),
+    Column("hive", TEXT, "HKEY_USERS registry hive"),
```

Comment:
May I ask you to explain a bit more here what does it mean?

Reply:
Sure! I'll add more detail.

Reply:
Also curious - what about calling this `registry_hive`? I understand how common the term Hive is for Windows users, but it might be useful to have a bit more detail in the name for those not familiar?

Reply:
Done!